lib/lease-manager.js

```diff
@@ -167,13 +167,16 @@ class LeaseManager {
    *        consumer or just simple polling consumers.
    * @param {number} options.leaseAcquisitionInterval - The interval in milliseconds for how often
    *        to attempt lease acquisitions.
+   * @param {number} options.leaseAcquisitionRecoveryInterval - The interval in milliseconds for how often
```

Comment:
params should be listed in abc order.  leaseAcquisitionRecoveryInterval and leaseAcquisitionInterval both should be placed in their appropriate order